backend/src/libs/packages/token/token.package.ts

```diff
@@ -0,0 +1,24 @@
+import * as jose from 'jose';
+
+import { config, type IConfig } from '../config/config.js';
```

Comment:
If you have both - imported config, and retrieved from `constructor` parameters - which of them is used, and where?
```suggestion
import { config, type IConfig } from '../config/config.js';
```